src/containers/my-cooperations/cooperation-details/CooperationDetails.styles.ts

```diff
@@ -7,9 +7,16 @@ export const styles = {
       typography: TypographyVariantEnum.H5,
       mb: '24px',
       mt: '16px',
-      width: { sm: '100%', md: '70%' }
+      width: { sm: '100%', md: '100%' },
```

Comment:
Please remove `'md: 100%'`, because if you write `'sm: 100%'` it will be applied to all screen sizes (from `sm` to `xl `including `md`)